demos/storybook/stories/empty-state/within-a-card.tsx

```diff
@@ -0,0 +1,84 @@
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import AddIcon from '@material-ui/icons/Add';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import HelpOutline from '@material-ui/icons/HelpOutline';
+import Typography from '@material-ui/core/Typography';
+import createStyles from '@material-ui/core/styles/createStyles';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import { Theme } from '@material-ui/core/styles/createMuiTheme';
+import useTheme from '@material-ui/core/styles/useTheme';
+import Button from '@material-ui/core/Button';
+import { EmptyState } from '@pxblue/react-components';
+import { action } from '@storybook/addon-actions';
+import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types';
+import { getLeftToRightIconTransform } from '../../src/utils';
+import React, { useState } from 'react';
+
+import * as Colors from '@pxblue/colors';
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        accordionPanel: {
+            margin: '0 auto',
+            width: '392px',
+        },
+        accordionRoot: {
+            '& .MuiAccordionSummary-root': {
+                borderBottom: `1px solid ${theme.palette.divider}`,
+                height: '48px',
+                minHeight: '48px',
+            },
+        },
+        accordionHeaderTitile: {
+            color: `${Colors.blue[500]}`,
```

Comment:
> `theme.palette.primary.main` is `#3f51b5`.

This seems to be working whenever I load it up; primary is blue[500].

![image](https://user-images.githubusercontent.com/6538289/136793308-cc4b5baa-2c4f-4de4-950b-50dc724da6e5.png)

Reply:
The useTheme hook is grabbing the wrong theme here. Every other color is hardcoded in the react storybook. So, I imagine there is some issue with using our themes within storybook. I'm not sure why yet, but I would recommend just hardcoding this value since there isn't a single reference to our colors via the theme in all the other stories. Feel free to try to change my mind ;)

Reply:
Take a look at this branch, I've made a fix where the makeStyles function uses a nested fn that accepts a theme prop.  This seems to work with light & dark themes, but I'm not entirely sure why. 

https://github.com/pxblue/react-component-library/blob/feature/fix-theme-in-story/demos/storybook/stories/empty-state/within-a-card.tsx

Reply:
@joebochill @daileytj @bkarambe  react expertise required

Reply:
Makes sense that that would work at the individual story level since you're just importing our theme manually instead of using the useTheme hook. Not sure you need to run that nested function inside of the makeStyles since you're passing in the appropriate theme.

Reply:
I thought so too but the top-level theme was not the `blue` or `blueDark` theme that was expected; it didn't use the blue[500] palette.